webhook/resources/volume/mutator.go

```diff
@@ -212,6 +212,14 @@ func (v *volumeMutator) Create(request *admission.Request, newObj runtime.Object
 	}
 	patchOps = append(patchOps, fmt.Sprintf(`{"op": "replace", "path": "/spec/engineImage", "value": "%s"}`, defaultEngineImage))
 
+	if volume.Spec.BackupCompressionMethod == "" {
+		defaultCompressionMethod, _ := v.ds.GetSettingValueExisted(types.SettingNameBackupCompressionMethod)
```

Comment:
Do we have a validation for setting value?

Reply:
Added the validation. Thank you.